charts/jaeger-v2/values.yaml

```diff
@@ -0,0 +1,62 @@
+service:
```

Comment:
this file has a generic name. What will happen when you need to support Cassandra, which requires different configuration?